scripts/dev

```diff
@@ -5,6 +5,26 @@ import consola from 'consola'
 
 import Package from './package.js'
 
+const useCjs = new Set([
+  '@nuxt/cli'
+])
+
+const stub = {
+  es: `export * from '../src/index'`,
```

Comment:
Why don't just add a shebang to the first line all files:
```
#! /usr/bin/env node -r esm
```

Reply:
Does that work for you on a mac? When I do that on linux it says:
```
/usr/bin/env: ‘node -r esm’: No such file or directory
```

Reply:
Don't want to add more complexity